src/model.py

```diff
@@ -163,7 +163,9 @@ def to_misp(self) -> MISPObject:
 
         # url-specific attributes
         if self.scheme:
-            obj.add_attribute("url", self.pretty_url)
+            url = obj.add_attribute("url", self.pretty_url)
+            if url:
```

Comment:
Checked pymisp, looks like the return value is only None if passed value is None (or non-printable bytes, or empty string)

https://pymisp.readthedocs.io/en/latest/_modules/pymisp/mispevent.html#MISPObject.add_attribute

I think we can skip this check here (and in most other places probably)

Btw
```python
        if simple_value is not None:  # /!\ The value *can* be 0
            value['value'] = simple_value
```
I wonder what's the story behind this comment :D

Reply:
:ack: